packages/manager/modules/catalog-price/src/catalog-price.html

```diff
@@ -12,69 +64,109 @@
-            ></span>
-        </span>
+    <strong data-ng-if="::!$ctrl.price" data-translate="order_catalog_price_free">
     </strong>
-    <strong
-        data-ng-if="::$ctrl.isUSFormat()"
-        data-ng-class="{ 'd-block': $ctrl.block }"
-    >
-        <span data-ng-bind="::$ctrl.getPriceText($ctrl.price)"></span>
-        <span data-ng-if="$ctrl.interval">
-            &#47;
-            <span
-                data-translate="{{:: 'order_catalog_price_interval_' + $ctrl.interval}}"
-            ></span>
+</div>
+
+<!--CASE CDN UPGRADE (No CDN)-->
+<span data-ng-if="$ctrl.cdnCase === 'without'">
+    <span data-ng-if="::$ctrl.price">
+        <span>
+            <strong data-ng-class="{ 'd-block': $ctrl.block }">
+                <span
+                    data-translate="{{:: 'order' + $ctrl.getKeyPartToConcat() + 'catalog_price_tax_excl_label'}}"
+                    data-translate-values="{ price: $ctrl.getPriceText($ctrl.price) }"
+                ></span>
+                <span data-ng-if="$ctrl.interval">
+                    &#47;
+                    <span
+                        data-translate="{{:: 'order_catalog_price_interval_' + $ctrl.interval}}"
+                    ></span>
+                </span>
+                <span
+                    data-translate="{{:: 'order' + $ctrl.getKeyPartToConcat() + 'catalog_price_tax_incl_label'}}"
+                    data-translate-values="{ price: $ctrl.getPriceText($ctrl.price + $ctrl.tax) }"
+                >
+                </span>
+            </strong>
         </span>
-    </strong>
+    </span>
+</span>
+
+<!--CASE CDN PAYABLE (CDN1 payed)-->
+<span data-ng-if="$ctrl.cdnCase === 'payable'">
+    <span data-ng-if="::$ctrl.price">
+        <span>
+            <strong data-ng-class="{ 'd-block': $ctrl.block }">
+                <span
+                    data-translate="{{:: 'order' + $ctrl.getKeyPartToConcat() + 'catalog_price_tax_incl_label'}}"
+                    data-translate-values="{ price: $ctrl.getPriceText($ctrl.cdn1Price.price + $ctrl.cdn1Price.tax) }"
+                >
+                </span>
+                <span data-ng-if="$ctrl.interval">
+                    &#47;
+                    <span
+                        data-translate="{{:: 'order_catalog_price_interval_' + $ctrl.interval}}"
+                    ></span>
+                </span>
+                <span
+                    data-translate="hosting_cdn_order_customer_cdn_payable_step1_price_to"></span>
+                <span
+                    data-translate="{{:: 'order' + $ctrl.getKeyPartToConcat() + 'catalog_price_tax_incl_label'}}"
+                    data-translate-values="{ price: $ctrl.getPriceText($ctrl.price + $ctrl.tax) }"
+                ></span>
+            </strong>
+        </span>
+    </span>
+</span>
+
+<!--CASE CDN INCLUDE (CDN1 included)-->
+<span data-ng-if="$ctrl.cdnCase === 'included'">
+    <span data-ng-if="::$ctrl.price">
```

Comment:
Combine the two conditions in one element

Reply:
Nice and changed